lib/tasks/thanks.rake

```diff
@@ -0,0 +1,9 @@
+task :thanks => :environment do
+  @messages = Message.all
```

Comment:
This should use a `find_each` with a block so that it doesn't try to load all 2.7 million + messages into memory.